### PR TITLE
swf-blueprint: add running-qemu-smoketests.rst

### PR DESCRIPTION
**Pull request by TSAnsari**
Added how to article for running tests on QEMU

Signed-off-by: Tariq Ansari <tansari@luxoft.com>

### CODE REVIEW

File: docs/articles/baseplatform/running-qemu-smoketests.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+.. _running-qemu-smoketests:
+
+Running smoke tests on QEMU
+===========================
+
+To run smoke tests on QEMU target, proceed as follows,
```
Comment (sashko) [inline docs/articles/baseplatform/running-qemu-smoketests.rst]: in QEMU

Comment (sashko) [reply docs/articles/baseplatform/running-qemu-smoketests.rst]: also, it seems to be officially called "image tests": https://wiki.yoctoproject.org/wiki/Image_tests

File: docs/articles/baseplatform/running-qemu-smoketests.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+.. code-block:: bash
+
+	# Bitbake core-image-pelux-minimal-dev for qemu-x86-64-nogfx variant
```
Comment (sashko) [inline docs/articles/baseplatform/running-qemu-smoketests.rst]: s/variant/target/

File: docs/articles/baseplatform/running-qemu-smoketests.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	# Make sure we have the required utilities
+	bitbake qemu-helper-native
+
+	# Set up networking for qemu
+	sudo ../sources/poky/scripts/runqemu-gen-tapdevs 1000 1000 4 tmp/sysroots-components/x86_64/qemu-helper-native/usr/bin
+
+	# Give qemu permissions to access tun interfaces
+	sudo setcap cap_net_admin+ep $(find tmp/work/ -name qemu-system-x86_64 | grep qemu-helper-native)
+
+	# Start smoke tests
```
Comment (sashko) [inline docs/articles/baseplatform/running-qemu-smoketests.rst]: Run tests

File: docs/articles/baseplatform/running-qemu-smoketests.rst (hunk 1 above)
Comment (TSAnsari) [reply docs/articles/baseplatform/running-qemu-smoketests.rst]: > in QEMU

:) I read it similar to as running "on NUC target" or "on RPI target".

File: docs/articles/baseplatform/running-qemu-smoketests.rst (hunk 2 above)
Comment (TSAnsari) [reply docs/articles/baseplatform/running-qemu-smoketests.rst]: qemu-x86-64-nogfx seems to be a variant of QEMU target.

File: docs/articles/baseplatform/running-qemu-tests.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Running tests on QEMU
+=====================
+
+To run tests on QEMU target, proceed as follows,
+
+.. code-block:: bash
+
+	# Bitbake core-image-pelux-minimal-dev for qemu-x86-64-nogfx variant
+	TEMPLATECONF=`pwd`/sources/meta-pelux/conf/variant/qemu-x86-64-nogfx source sources/poky/oe-init-build-env build
```
Comment (fhajredini) [inline docs/articles/baseplatform/running-qemu-tests.rst]: typo: /sources/meta-pelux/conf/variant/qemu-x86-64_nogfx

File: docs/articles/baseplatform/running-qemu-tests.rst [hunk 5]
```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+.. _running-qemu-tests:
+
+Running tests on QEMU
+=====================
+
```
Comment (fhajredini) [inline docs/articles/baseplatform/running-qemu-tests.rst:7]: Probably adding some description on where the tests are coming from is good to have.. maybe a brief elaboration of the testimage and the tests on local.conf

**Review by fhajredini — CHANGES_REQUESTED** [review]
Fix typo on the variant name

**Comment by TSAnsari** [comment]
> Fix typo on the variant name

Fixed!

File: docs/articles/baseplatform/running-qemu-smoketests.rst (hunk 2 above)
Comment (sashko) [reply docs/articles/baseplatform/running-qemu-smoketests.rst]: it's a target platform, isn't it?

Comment (sashko) [reply docs/articles/baseplatform/running-qemu-smoketests.rst]: variants are minimal ant qtauto

Comment (TSAnsari) [reply docs/articles/baseplatform/running-qemu-smoketests.rst]: Fixed

**Comment by TSAnsari** [comment]
@fhajredini @sashko Any more comments on this? Will merge if there are no more comments.

**Review by sashko — APPROVED** [review]

**Pull request merged**